ui/ducks/swaps/swaps.js

```diff
@@ -746,7 +747,6 @@ export const fetchQuotesAndSetQuoteState = (
     const hardwareWalletType = getHardwareWalletType(state);
     const networkAndAccountSupports1559 =
       checkNetworkAndAccountSupports1559(state);
-    const smartTransactionsOptInStatus = getSmartTransactionsOptInStatus(state);
```

Comment:
Why repeat the selector a few times below instead of replacing this with something like `const smartTransactionsOptInStatusForMetrics = getSmartTransactionsOptInStatusForMetrics(state)`?

Reply:
I didn’t use a new variable because I felt it slightly reduced readability and wasn’t necessary for consistency with the rest of the file. Plus, the selector is memoized, so there’s no performance impact.